src/main/cpp/mechanisms/DragonTale/GrabAlgaeReefState.cpp

```diff
@@ -76,6 +76,6 @@ bool GrabAlgaeReefState::AtTarget()
 bool GrabAlgaeReefState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	return false;
-	// return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXAMPLE_MECH_FORWARD));
+
```

Comment:
At each OR statement make a new line for readability

Current State is ScoreCoral AND Human Player Button Pressed AND Algae Sensor = False AND Coral In Sensor = False AND Coral Out Sensor = False, they need to be NOTs (!m_mechanism->GetAlgaeSensorState())

Delete the second Algae Mode AND Human Player Button repeated line